verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -480,6 +480,9 @@ class SourceTable extends React.Component<Props, SourceTableState> {
                                     <SourceRetrievalButton sourceId={row._id} />
                                 ),
                                 editable: 'never',
+                                hidden: !this.props.user?.roles.includes(
```

Comment:
This was removed because you can only access the Sources table if you are a curator